### PR TITLE
[DM-5637] Update README.md and remove unused dependency.

### PR DESCRIPTION
**Pull request by jmatt**
Update README.md, LICENSE and remove unused dependency.

	modified:   LICENSE
	modified:   README.md
	modified:   setup.py

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,10 +6,34 @@ Github TravisCI and Flake8 and update automation for LSST DM SQuaRE.
...
+* `github-travis` - Enable TravisCI webhook for repository(ies).
+
+* `github-update` - Create a branch and update GitHub repository(ies).
+
+* `github-protect-travis` - Add Github branch protection, TravisCI webhooks and flake8 support through one script.
+
+The following is an example that adds Github branch protection, enables TravisCI then creates a commit and pull request with a .travis.yml and setup.cfg that runs flake8. This example is for [lsst.utils](https://github.com/lsst/utils).
+
+```
+github-protect --owner lsst -repo utils --branch_name master
```
Comment (timj) [inline README.md]: Single `-` here not `--`?

File: README.md [hunk 2]
```diff
@@ -6,10 +6,34 @@ Github TravisCI and Flake8 and update automation for LSST DM SQuaRE.
...
+github-travis --owner lsst --repo utils
+github-update --owner lsst --repo utils --task stack -branch_name tickets/DM-5637 --commit_message "[DM-5637] Add .travis.yml and setup.cfg to run flake8." --pull --pull_message "[DM-5637] Add .travis.yml and setup.cfg to run flake8."
```
Comment (timj) [inline README.md]: It seems odd that all the command line options have `--` but then there is `-branch_name`.

**Comment by timj** [comment]
Don't forget to mention the auth token stuff.

File: README.md (hunk 2 above)
Comment (jmatt) [reply README.md]: The console scripts support both double and single dash options. For this I standardized on all double dash options, but could mix it up. I fixed the two single dash options you mentioned and made them double dashes.

File: README.md (hunk 1 above)
Comment (jmatt) [reply README.md]: fixed. That's a typo. It's either -r or --repo.

File: README.md (hunk 2 above)
Comment (timj) [reply README.md]: Some seem to support single and others require double. Use `--` consistently.

Comment (jmatt) [reply README.md]: All the single dashes use one letter. Per python's argparse conventions. Those were typos. https://github.com/lsst-sqre/sqre-gtf/blob/master/gtf/base.py#L50:L68

**Review by timj — APPROVED** [review]

**Pull request merged**